src/net/ifaddrs.c

```diff
@@ -6,6 +6,7 @@
 #include <unistd.h>
 #include <sys/socket.h>
 #define __USE_MISC 1   /**< Use MISC code */
+#define _BSD_SOURCE 1
```

Comment:
this file is already present in main HEAD.

This patch has to be rebased on top of HEAD.